convert.go

```diff
@@ -137,26 +136,50 @@ func componentConverter(specVersion SpecVersion) func(*Component) {
 		if specVersion < SpecVersion1_5 {
 			c.ModelCard = nil
 			c.Data = nil
-
-			if c.Evidence != nil {
-				c.Evidence.Identity = nil
-				c.Evidence.Occurrences = nil
-				c.Evidence.Callstack = nil
-			}
 		}
 
 		if !specVersion.supportsComponentType(c.Type) {
 			c.Type = ComponentTypeApplication
 		}
+
 		convertExternalReferences(c.ExternalReferences, specVersion)
 		convertHashes(c.Hashes, specVersion)
 		convertLicenses(c.Licenses, specVersion)
+		convertEvidence(c.Evidence, specVersion)
+
 		if !specVersion.supportsScope(c.Scope) {
 			c.Scope = ""
 		}
 	}
 }
 
+func convertEvidence(ev *Evidence, specVersion SpecVersion) {
+	if ev == nil {
+		return
+	}
+
+	if specVersion < SpecVersion1_3 {
+		ev = nil
+	}
+
+	if specVersion < SpecVersion1_5 {
+		ev.Identity = nil
+		ev.Occurrences = nil
+		ev.Callstack = nil
+	}
```

Comment:
We should return here to prevent a `nil` panic further down.

```suggestion
		return
	}
```